pricelist_brand/tests/test_product_pricelist.py

```diff
@@ -112,26 +112,21 @@ def test_calculation_price_of_products_pricelist(self):
         context.update({"pricelist": self.pricelist.id, "quantity": 1})
 
         # Check sale price of branded product
-        product_with_context = self.product.with_context(context)
-        self.assertEqual(
-            float_compare(
-                product_with_context.price,
-                (
-                    product_with_context.lst_price
-                    - product_with_context.lst_price * (0.10)
-                ),
-                precision_digits=2,
-            ),
-            0,
+        product = self.product.with_context(context)
+        price = self.pricelist._get_product_price(product, quantity=1.0)
```

Comment:
`product.price` (with the correct context) does not return the pricelist price anymore ?